lib/deprecations.js

```diff
@@ -2,6 +2,14 @@
 
 const warning = require('process-warning')()
 
-warning.create('LdapjsFilterWarning', 'LDAP_FILTER_DEP_001', 'parse is deprecated. Use the parseString function instead.')
+const clazz = 'LdapjsFilterWarning'
+
+warning.create(clazz, 'LDAP_FILTER_DEP_001', 'parse is deprecated. Use the parseString function instead.')
+
+warning.create(clazz, 'LDAP_SUBSTRING_FILTER_DEP_001', 'subInitial is deprecated. Use initial instead.')
+
+warning.create(clazz, 'LDAP_SUBSTRING_FILTER_DEP_002', 'subAny is deprecated. Use any instead.')
+
+warning.create(clazz, 'LDAP_SUBSTRING_FILTER_DEP_003', 'subFinal is deprecated. Use final instead.')
```

Comment:
The error code is specific to the package, not the individual features of the package. So:

```suggestion
warning.create(clazz, 'LDAP_FILTER_DEP_002', 'subInitial is deprecated. Use initial instead.')

warning.create(clazz, 'LDAP_FILTER_DEP_003', 'subAny is deprecated. Use any instead.')

warning.create(clazz, 'LDAP_FILTER_DEP_004', 'subFinal is deprecated. Use final instead.')
```

Reply:
Okay! I updated the error codes.